### ISSUE TITLE
Accessibility: Add scroll back to top button

### ISSUE THREAD

**Issue by D-Antonelli** [reactjs/reactjs.org#3679]
When I go to a topic section, I need to scroll all the way up to go to the top of the page to access the navigation menu. 

For example
1. Go to [Hooks FAQ](https://reactjs.org/docs/hooks-faq.html)
2. Click a navigation [button](https://reactjs.org/docs/hooks-faq.html#which-versions-of-react-include-hooks)

## The current behavior
I need to scroll all the way up to go to the table of contents.

## Expected behaviour
Click a scroll to the top button for instant access to the navigation menu like this [one](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_scroll_to_top).

**Comment by grimxreaper** [comment]
I can see the need for a scroll to top button regardless, I think it's a great idea, but for reference, I can still see the navigation menu even when I scroll down, pic included. _Is this what you see too?_ Either way, I'd love to take this issue on as my first contrib to this repo if possible! 🙂

As you can see, even if I have scrolled down on the page, the navbar is still visible. 
<img width="1319" alt="Screen Shot 2021-05-13 at 10 11 49 AM" src="https://user-images.githubusercontent.com/64915527/118137828-a30de400-b3d3-11eb-9dd5-c9223d2aeb01.png">

**Comment by D-Antonelli** [comment]
Hi @grimxreaper! I just noticed my statement about navigation needs correction. What I meant by navigation is indeed the "FAQ" section. 
\
The FAQ section starts with "Adoption Strategy". When I click a link, it takes me to the related section on the same page. Everything works fine until this point. However, when I want to go back to the FAQ section because I want to check another question,  I need to scroll up all the way back. \
\
![Capture](https://user-images.githubusercontent.com/49251832/118232664-d0689980-b488-11eb-8acd-49e5be6653d9.JPG)
![Capture2](https://user-images.githubusercontent.com/49251832/118232746-efffc200-b488-11eb-83c6-590760fd1bd1.JPG)

**Comment by grimxreaper** [comment]
Ah I see @D-Antonelli thank you for clarifying! Even in that case I guess a scroll to the top button would be the fix needed to navigate back to the FAQ section. Could I try a stab at implement this button @eps1lon ? I'm it would need to be on each page of the website, so there should be a way to implement it in such a way that it becomes available on each page?

**Comment by D-Antonelli** [comment]
> Even in that case I guess a scroll to the top button would be the fix needed to navigate back to the FAQ section. Does that sound like what you were imagining?

Yes, @grimxreaper! I am suggesting back to top button like this example:

[https://codepen.io/matthewcain/pen/ZepbeR](url)

**Comment by doiali** [comment]
@D-Antonelli the current active page (for example Hooks FAQ ) is highlighted in the right side menu
you can click that link to navigate back to top
you don't need to worry about browser history because clicking this link doesn't push history

**Comment by D-Antonelli** [comment]
@doiali I get your point. I see the right-hand sidebar maps Hooks section and show Hooks FAQ in bold while I am there. It is not a wrong practice but to improve things further, go to the top button is a more predictable way of navigating the user inside the page rather than checking the Hooks site map.
